docs/gantt.md

```diff
@@ -15,18 +15,6 @@ gantt
     Task in sec      :2014-01-12  , 12d
     another task      : 24d
 ```
-```mermaid
```

Comment:
Hey, here you deleted the sample rendered graph of the syntax shown above. Is that for any particular reason?

Reply:
Sorry. I thought it was a duplication. I will revert back the change

Reply:
Not a problem. That second one gets rendered because it starts with: 

       ```mermaid